### PR TITLE
just name

### PR DESCRIPTION
**Pull request by Diaaana03**

### CODE REVIEW

**Review by cris-valente — CHANGES_REQUESTED** [review]
Please refer to our [Figma design](https://www.figma.com/file/Z1sSZYngU1YoNi6HnfktuP/Voyage-47---Team-3?type=design&node-id=262-2677&mode=design&t=IX8ds88OORKeuXoq-0), Diana. I made a few comments on each line

File: styles.css [hunk 1]
```diff
@@ -470,6 +470,15 @@ body {
...
   margin-left: 28px;
   margin-top: 27px;
+  position: absolute;
+}
+
+.name {
+  display: flex;
+  justify-content: center;
+  color: var(--white);
+  font-size: 1.4rem;
```
[the remark this answers is not in the document]
Comment (Diaaana03) [reply styles.css]: done

**Review by Diaaana03 — COMMENTED** [review]
done

**Review by Morais-C — APPROVED** [review]

**Review by Morais-C — APPROVED** [review]

**Review by cris-valente — APPROVED** [review]
Nice! Approved.

**Pull request merged**